tests/test_create_people_page.py

```diff
@@ -0,0 +1,22 @@
+from pages.CreateUserPage import CreateUserPage
+from tests.locators_people_page import URLLocators
+from tests.locators_people_page import InputFormNewUser
+
+
+class TestCreatePeoplePage:
+
+    def test_create_people(self):
+        driver = CreateUserPage(self.driver)
+        driver.go_to_page(URLLocators.URL_PEOPLE_CREATE)
+
+        driver.enter_username("user1")
+        driver.enter_password("123")
+        driver.enter_confirm_password("123")
+        driver.enter_fullname("User User")
+        driver.enter_email("user@gmail.com")
+        driver.click_create_button()
+
+        driver.go_to_page(URLLocators.URL_PEOPLE)
+        field = driver.get_element_text(InputFormNewUser.FIELD_NEW_USER).split(' ')
+
+        assert field[0] == "user1"
```

Comment:
Good TC, I like it. But what if we run it several times? It will create several users with the same name or it will fail?
I would suggest using a random name. in this case we avoid duplicating.
Plus, you are hardcoding data in lines 12-16, which is not a good practice.

Reply:
If you invoke several times test Create, it will be pass, because assert finds row in the table with id='person-{Your_user_name}' and if the user exists this row is present, but Jenkins doesn't create the same user, and we can see alert "User name is already taken". By the way, now I use hardcode because this is a condition in my test case, but in the future, it will be a theme for refactoring

Reply:
I made a random choice of username